translations/xblock-drag-and-drop-v2/drag_and_drop_v2/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-08-24 00:24+0000\n"
```

Comment:
verify that a bug exists, resolved in https://github.com/openedx/i18n-tools/pull/136